pocs/utils/google/storage.py

```diff
@@ -317,6 +317,8 @@ def upload_observation_to_bucket(pan_id,
     """
     assert os.path.exists(dir_name)
     assert re.match(r'PAN\d\d\d', pan_id) is not None
+    # Note: this just makes sure the command exists, it is called from bash file.
```

Comment:
Doesn't of course ensure, since it is only an assert. If you want these asserts to be meaningful, how about converting them to raise an exception?